Source/Fuse.Scripting.JavaScript/ThreadWorker.uno

```diff
@@ -76,15 +73,8 @@ namespace Fuse.Scripting.JavaScript
 
 		void RunInner()
 		{
-			try
-			{
-				_context = Fuse.Scripting.JavaScript.JSContext.Create();
-				UpdateManager.AddAction(CheckAndThrow);
-			}
-			finally
-			{
-				_ready.Set();
-			}
+			_context = Fuse.Scripting.JavaScript.JSContext.Create();
+			UpdateManager.AddAction(CheckAndThrow);
```

Comment:
This call to `AddAction` looks like an error. `RunInner` is being called from the thread, right?  The only safe UpdateManager call form a non-UI thread is `PostAction`.

Reply:
@mortoray: This was already the case, though.

Reply:
I have some changes in this area, I'll fix the problem as part of that.

Reply:
Interesting, I'll open an issue for this